javascript/operators/object_initializer.json

```diff
@@ -0,0 +1,277 @@
+                "version_added": true
+              },
+              "edge": {
+                "version_added": true
+              },
+              "edge_mobile": {
+                "version_added": true
+              },
+              "firefox": {
+                "version_added": "34"
+              },
+              "firefox_android": {
+                "version_added": "34"
+              },
+              "ie": {
+                "version_added": false
+              },
+              "ie_mobile": {
+                "version_added": false
+              },
+              "nodejs": {
+                "version_added": true
+              },
+              "opera": {
+                "version_added": "34"
+              },
+              "opera_android": {
+                "version_added": "34"
+              },
+              "safari": {
+                "version_added": "7.1"
+              },
+              "safari_ios": {
+                "version_added": "7.1"
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "shorthand_property_names": {
+          "__compat": {
+            "description": "Shorthand property names",
+            "support": {
+              "webview_android": {
+                "version_added": true
+              },
+              "chrome": {
+                "version_added": true
+              },
+              "chrome_android": {
+                "version_added": true
+              },
+              "edge": {
+                "version_added": true
+              },
+              "edge_mobile": {
+                "version_added": true
+              },
+              "firefox": {
+                "version_added": "33"
+              },
+              "firefox_android": {
+                "version_added": "33"
+              },
+              "ie": {
+                "version_added": false
+              },
+              "ie_mobile": {
+                "version_added": false
+              },
+              "nodejs": {
+                "version_added": true
+              },
+              "opera": {
+                "version_added": "34"
+              },
+              "opera_android": {
+                "version_added": "34"
+              },
+              "safari": {
+                "version_added": "9"
+              },
+              "safari_ios": {
+                "version_added": "9"
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "shorthand_method_names": {
+          "__compat": {
+            "description": "Shorthand method names",
+            "support": {
+              "webview_android": {
+                "version_added": "42"
```

Comment:
There is a discrepency between Chrome 42 and Opera 34 (that is based on Chrome 47). Worth double-checking these values.

Reply:
Lets put 47 for now. We'll accept correction contributions